### PR TITLE
Increasing test coverage of logger package

### PR DESCRIPTION
**Pull request by mativm02**
## Description

I've created a unit test for GetLogger() function. I've also deleted some unnecessary lines of code. Whenever TYK_LOGLEVEL has an invalid value (empty or a not valid string), the log level will be set as "info". That's why we don't need to add an "info" case in the switch sentence, we can just keep it as default.
## Screenshots (if appropriate)

**Before:**
212693397-5371fa31-f0c5-4df0-bbbb-eb74eb4f7cd2

**After:**
212693338-47593ddc-8ac9-4481-acf9-69220a804747

## Types of changes

- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)

## Checklist

- [ ] Make sure you are requesting to **pull a topic/feature/bugfix branch** (right side). If pulling from your own
      fork, don't request your `master`!
- [ ] Make sure you are making a pull request against the **`master` branch** (left side). Also, you should start
      *your branch* off *our latest `master`*.
- [ ] My change requires a change to the documentation.
  - [ ] If you've changed APIs, describe what needs to be updated in the documentation.
- [ ] I have updated the documentation accordingly.
- [ ] Modules and vendor dependencies have been updated; run `go mod tidy && go mod vendor`
- [ ] I have added tests to cover my changes.
- [ ] All new and existing tests passed.
- [ ] Check your code additions will not fail linting checks:
  - [ ] `go fmt -s`
  - [ ] `go vet`

### CODE REVIEW

File: logger/init.go [hunk 1]
```diff
@@ -24,17 +24,12 @@ func GetFormatterWithForcedPrefix() *prefixed.TextFormatter {
 
 func GetLogger() *logrus.Logger {
 	level := os.Getenv("TYK_LOGLEVEL")
-	if level == "" {
```
Comment (tbuchaillot) [inline logger/init.go:27]: @mativm02 why is this removed?

Comment (mativm02) [reply logger/init.go:27]: Because "info" is the default value. If the user sends an empty string, we don't need to convert it to "info", since it will join the "default" case in the switch sentence, which set the log level to "InfoLevel".
That's also the reason why I've removed `case "info":` in the switch sentence.

File: logger/init_test.go [hunk 2]
```diff
@@ -51,3 +51,47 @@ func TestFormatterWithForcedPrefixFileOutput(t *testing.T) {
...
+		{
+			name:          "debug",
+			env:           "debug",
+			expectedLevel: logrus.DebugLevel,
+		},
+	}
+
+	for _, tt := range tests {
+		t.Run(tt.name, func(t *testing.T) {
+			os.Setenv("TYK_LOGLEVEL", tt.env)
```
Comment (tbuchaillot) [inline logger/init_test.go:90]: @mativm02 missed the defer of unset of the env after the test

File: logger/init.go (hunk 1 above)
Comment (tbuchaillot) [reply logger/init.go:27]: makes sense!

**Review by tbuchaillot — CHANGES_REQUESTED** [review]

File: logger/init_test.go (hunk 2 above)
Comment (mativm02) [reply logger/init_test.go:90]: @tbuchaillot I will add it! thanks

**Review by tbuchaillot — APPROVED** [review]

**Review by sredxny — APPROVED** [review]
lgtm
